_layouts/base.html

```diff
@@ -87,75 +89,78 @@
-              class="usa-logo__img"
-              alt="Login.gov"
-              role="img">
-            <em class="usa-logo__text">Developers</em>
-          </a>
-        </div>
-        <button class="usa-menu-btn">Menu</button>
-      </div>
-      <nav role="navigation" class="usa-nav border-top-none">
-        <div class="usa-nav__inner">
-          <button class="usa-nav__close">
-            <img src="{{ site.baseurl }}/assets/img/close.svg" alt="Close" role="img">
-          </button>
-          <div class="usa-nav__secondary">
-            <ul class="usa-nav__secondary-links"></ul>
-            <section aria-label="Search component">
-              {% include search.html id="header-nav" %}
-            </section>
+      <header class="usa-header usa-header--extended" role="banner">
+          <div class="usa-navbar">
+            <div class="usa-logo" id="basic-logo">
+              <a href="{{ site.baseurl }}/" title="Home" aria-label="Home">
+                <img src="{{ site.baseurl }}/assets/img/login-gov-logo.svg"
+                  class="usa-logo__img"
+                  alt="Login.gov"
+                  role="img">
+                <em class="usa-logo__text">Developers</em>
+              </a>
+            </div>
+            <button class="usa-menu-btn">Menu</button>
           </div>
-          <ul class="usa-nav__primary usa-accordion">
-            {% include nav/list.html
-              links = site.data.nav.primary
-              li_class = 'usa-nav__primary-item'
-              subnav_ul_class = 'desktop:display-none usa-sidenav__sublist'
-            %}
-          </ul>
-        </div>
-      </nav>
-    </header>
+          <nav role="navigation" class="usa-nav border-top-none">
+            <div class="usa-nav__inner">
+              <button class="usa-nav__close">
+                <img src="{{ site.baseurl }}/assets/img/close.svg" alt="Close" role="img">
+              </button>
+              <div class="usa-nav__secondary">
+                <ul class="usa-nav__secondary-links"></ul>
+                <section aria-label="Search component">
+                  {% include search.html id="header-nav" %}
+                </section>
+              </div>
+              <ul class="usa-nav__primary usa-accordion">
+                {% include nav/list.html
+                  links = site.data.nav.primary
+                  li_class = 'usa-nav__primary-item'
+                  subnav_ul_class = 'desktop:display-none usa-sidenav__sublist'
+                %}
+              </ul>
+            </div>
+          </nav>
+      </header>
+      {% if page.sidenav %}
+      <header aria-label="hero" class="usa-dark-background usa-section--dark">
+        <section class="usa-section">
+          <div class="grid-container">
+            <div class="grid-row">
+              <div class="grid-col-12">
+                    <h1 class="usa-display title">{{ page.title }}</h1>
```

Comment:
Are there `title` class styles defined somewhere? I can't find them.

```suggestion
                    <h1 class="usa-display">{{ page.title }}</h1>
```